core/trino-spi/src/test/java/io/trino/spi/type/TestTimeZoneKey.java

```diff
@@ -236,7 +236,7 @@ public void testRoundTripSerialization()
         }
     }
 
-    public void assertTimeZoneNotSupported(String zoneId)
+    private void assertTimeZoneNotSupported(String zoneId)
```

Comment:
This probably need to come before `Rename trino-testng-services to trino-testing-services`

Reply:
not really. It becomes a problem after adding trino-testng-services as a dependency of trino-spi, which is done as part of `Uniform JMH benchmark initialization`